moveit_simple/src/robot.cpp

```diff
@@ -417,42 +417,33 @@ bool Robot::getJointSolution(const Eigen::Affine3d& pose, double timeout, const
     local_seed = getJointState();
   }
 
-  return getIK(pose, local_seed, joint_point, timeout);
+  return getIK(pose, local_seed, joint_point, timeout, 1, limit_joint_windup_);
```

Comment:
```suggestion
  const unsigned int attempts = 1;
  return getIK(pose, local_seed, joint_point, timeout, attempts, limit_joint_windup_);
```
improves documentation

Reply:
Probably makes sense. The `attempts` parameter will be removed when we're switching to the master branch, though.